code/modules/reagents/dispenser/dispenser2.dm

```diff
@@ -99,6 +99,8 @@
 
 	if(href_list["dispense"])
 		var/label = href_list["dispense"]
+		if(user.skill_check(core_skill, SKILL_BASIC) && prob(25))
```

Comment:
```suggestion
		if(!user.skill_check(core_skill, SKILL_BASIC) && prob(25))
```
What you did right now is make it randomly dispense if someone HAS basic skill or above.